lib/core/src/Cardano/Wallet.hs

```diff
@@ -1065,6 +1096,18 @@ data ErrStartTimeLaterThanEndTime = ErrStartTimeLaterThanEndTime
     , errEndTime :: UTCTime
     } deriving (Show, Eq)
 
+data ErrPostTransaction
+    = ErrPostTransactionSign ErrSignTx
+    | ErrPostTransactionSubmit ErrSubmitTx
+    deriving (Show, Eq)
+
+data ErrMigrationWallet
+    = ErrMigrationWalletNoSuchWallet ErrNoSuchWallet
+    | ErrMigrationWalletSeqWalletInSource
+    | ErrMigrationWalletRndWalletInDestination
```

Comment:
These would probably need to have wallet ids as parameter to make for nicer error, no :thinking: ?

Reply:
agree, added `WalletId`